Base/res/keymaps/sl.json

```diff
@@ -1,6 +1,6 @@
-    "shift_map": [ "", "\u001B", "!", "\"", "#", "$", "%", "&", "/", "(", ")", "=", "?", "*", "\b", "\t",  "Q", "W", "E", "R", "T", "Z", "U", "I", "O", "P", "Š", "Đ", "\n", "", "A", "S", "D", "F", "G", "H", "J", "K", "L", "Č", "Ć", "¨", "", "Ž", "Y", "X", "C", "V", "B", "N", "M", ";", ":", "_", "", "*", "", " ", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "7", "8", "9", "-", "4", "5", "6", "+", "1", "2", "3", "0", ",", "", ">", "", "", ""], 
-    "alt_map":   [ "", "\u001B", "1",  "2", "3", "4", "5", "6", "7", "8", "9", "0", "'", "+", "\b", "\t",  "q", "w", "e", "r", "t", "z", "u", "i", "o", "p", "š", "đ", "\n", "", "a", "s", "d", "f", "g", "h", "j", "k", "l", "č", "ć", "¸", "", "ž", "y", "x", "c", "v", "b", "n", "m", ",", ".", "-", "", "*", "", " ", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "7", "8", "9", "-", "4", "5", "6", "+", "1", "2", "3", "0", ",", "", "<", "", "", ""], 
+    "map":       [ "", "\u001B", "1",  "2", "3", "4", "5", "6", "7", "8", "9", "0", "'", "+", "\b", "\t",  "q", "w", "e", "r", "t", "z", "u", "i", "o", "p", "š", "đ", "\n", "", "a", "s", "d", "f", "g", "h", "j", "k", "l", "č", "ć", "¸", "", "ž", "y", "x", "c", "v", "b", "n", "m", ",", ".", "-", "", "*", "", " ", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "7", "8", "9", "-", "4", "5", "6", "+", "1", "2", "3", "0", ",", "", "<", "", "", ""],
+    "shift_map": [ "", "\u001B", "!", "\"", "#", "$", "%", "&", "/", "(", ")", "=", "?", "*", "\b", "\t",  "Q", "W", "E", "R", "T", "Z", "U", "I", "O", "P", "Š", "Đ", "\n", "", "A", "S", "D", "F", "G", "H", "J", "K", "L", "Č", "Ć", "¨", "", "Ž", "Y", "X", "C", "V", "B", "N", "M", ";", ":", "_", "", "*", "", " ", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "7", "8", "9", "-", "4", "5", "6", "+", "1", "2", "3", "0", ",", "", ">", "", "", ""],
+    "alt_map":   [ "", "\u001B", "1",  "2", "3", "4", "5", "6", "7", "8", "9", "0", "'", "+", "\b", "\t",  "q", "w", "e", "r", "t", "z", "u", "i", "o", "p", "š", "đ", "\n", "", "a", "s", "d", "f", "g", "h", "j", "k", "l", "č", "ć", "¸", "", "ž", "y", "x", "c", "v", "b", "n", "m", ",", ".", "-", "", "*", "", " ", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "7", "8", "9", "-", "4", "5", "6", "+", "1", "2", "3", "0", ",", "", "<", "", "", ""],
     "altgr_map": [ "", "\u001B", "~",   "", "^",  "",  "",  "", "`",  "",  "",  "",  "",  "", "\b", "\t", "\\", "|", "€",  "",  "",  "",  "",  "",  "",  "", "÷", "×", "\n", "",  "",  "",  "", "[", "]",  "",  "", "ł", "Ł",  "", "ß",  "", "", "¤",  "",  "",  "", "@", "{", "}", "§", "<", ">",  "", "", "*", "", " ", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "7", "8", "9", "-", "4", "5", "6", "+", "1", "2", "3", "0", ",", "",  "", "", "", ""]
-}
```

Comment:
Oh, great! I accidentally also fixed this. Maybe we should make `check-newlines-at-eof.py` also check `json` files. Or, at least, keymap files.